UnboundLocalError: local variable 'cmd_name' referenced before assignment

Package version (if known): 0.2.4

## Describe the bug

```
    def create(self):
        """Any POST business logic."""
        action = resource_requestctx.route["action"]
        try:
            cmd_name = self.config.action_commands[action]
            cmd_func = getattr(self.service, cmd_name)
        except KeyError:
            raise ActionNotConfigured(action=action)
        except AttributeError:
>           raise CommandNotImplemented(cmd_name)
E           UnboundLocalError: local variable 'cmd_name' referenced before assignment
```


## Steps to Reproduce
1. Don't define `action_commands` for the `self.config` at all in the above 
2. run above `create()`

## Expected behavior
raise CommandNotImplemented with the cmd_name

--- a/invenio_drafts_resources/resources/draft_config.py
+++ b/invenio_drafts_resources/resources/draft_config.py
@@ -40,3 +40,6 @@ class DraftActionResourceConfig(ResourceConfig):
     response_handlers = {
         "application/json": RecordResponse(RecordDraftJSONSerializer())
     }
+    action_commands = {
+        "publish": "publish",
+    }

